goodcheck.yml

```diff
@@ -3,7 +3,7 @@ rules:
     pattern: 💪👽🚨
     message: Do you forget to delete `arglists` section?
     glob:
-      - "stdlib/**/*.rbs"
+      - "**/*.rbs"
```

Comment:
Nice catch. 👍 
Could you make it `stdlib/**/*.rbs` and `core/**/*.rbs`? The distributed RBS files are saved there.

Reply:
I've done it! (rebasing `master` and squashed to one commit)